pwndbg/ida.py

```diff
@@ -97,19 +97,13 @@ def available():
 
 
 def l2r(addr):
-    exe = pwndbg.elf.exe()
-    if not exe:
-        raise Exception("Can't find EXE base")
-    result = (addr - int(exe.address) + base()) & pwndbg.arch.ptrmask
-    return result
+    vaddr = pwndbg.vmmap.find(pwndbg.regs.pc).vaddr
+    return (addr - vaddr + base()) & pwndbg.arch.ptrmask
 
 
 def r2l(addr):
```

Comment:
The `L` and `R` stand for `local` and `remote` respectively, where `gdb` is "local" and IDA is "remote".  They are indeed to convert addresses between the real addresses that GDB sees, and whatever addresses IDA is using.  We could rebase the IDB, but it's expensive and leads to really weird errors, so we just adjust addresses on-the-fly.